Makefile

```diff
@@ -781,14 +785,10 @@ test-v8 test-v8-intl test-v8-benchmarks test-v8-all:
 	$(warning Use the git repo instead: $$ git clone https://github.com/nodejs/node.git)
 endif
 
-apidoc_dirs = out/doc out/doc/api out/doc/api/assets
+apidoc_dirs = out/doc out/doc/api
 skip_apidoc_files = doc/api/quic.md
 
-apidoc_sources = $(filter-out $(skip_apidoc_files), $(wildcard doc/api/*.md))
-apidocs_html = $(addprefix out/,$(apidoc_sources:.md=.html))
-apidocs_json = $(addprefix out/,$(apidoc_sources:.md=.json))
-
-apiassets = $(subst api_assets,api/assets,$(addprefix out/,$(wildcard doc/api_assets/*)))
+run-npm-ci = $(PWD)/$(NPM) ci
```

Comment:
We can probably get faster download times if we only install production dependencies

Reply:
Agreed.